conda/plugins/hookspec.py

```diff
@@ -208,3 +209,31 @@ def conda_auth_handlers():
                     auth_handler=EnvironmentHeaderAuth,
                 )
         """
+
+    @_hookspec
+    def conda_post_solves(self) -> Iterable[CondaPostSolve]:
+        """
+        Register post-solve functions in conda.
+
+        **Example:**
+
+        .. code-block:: python
+
+           from conda import plugins
+
+
+           def example_post_solve(
+               repodata_fn: str,
+               unlink_precs: tuple[PackageRecord, ...],
+               link_precs: tuple[PackageRecord, ...],
+           ):
+               print("post-solve action")
```

Comment:
Hmm, let's add a realistic example here, this is going to show up in the docs after all. Maybe printing the to-be-added and to-be-removed packages?

Reply:
see https://github.com/conda/conda/pull/13053/commits/85462f9e2b67b8028cc08c3955857736c33fae93